guide/index.md

```diff
@@ -38,41 +40,36 @@ Use your room manager URL to fetch peer token to get a new room:
 
 ```ts
 const response = await fetch(
-  `https://fishjam.io/api/v1/connect/*YOUR_ID*/room-manager/?roomName=*roomName*&participantName=*username*`,
+  `https://fishjam.io/api/v1/connect/*YOUR_ID*/room-manager/?roomName=*roomName*&peerName=*username*`
 );
 
 const { url, peerToken } = await response.json();
 ```
 
 ### 2. Join Room and start streaming
 
-:::danger
+:::warning
 
-If you want to use the camera, you must first request permission.
-Check [this `guide`](./react-native/installation#step-2-configure-app-permissions) for more information.
+If you want to use the camera, you must first request permission. Check
+[permission guide](./react-native/installation#step-2-configure-app-permissions) for more information.
 
 :::
 
 :::warning
 
-Keep in mind that this won't work on iOS Simulator.
+Keep in mind that this won't work on iOS Simulator, as Simulator can't access camera.
+
 :::
 
 To start streaming, you have to prepare your camera and join the room:
 
 ```tsx
-function StartStreamingButton({
-  roomName,
-  userName,
-}: {
-  roomName: string;
-  userName: string;
-}) {
+function StartStreamingButton({ roomName, userName }: { roomName: string; userName: string }) {
   const { prepareCamera } = useCamera();
 
   const startStreaming = useCallback(async () => {
     const response = await fetch(
-      `https://cloud.fishjam.work/api/v1/connect/*YOUR_ID*/room-manager/${roomName}/users/${userName}`,
+      `https://cloud.fishjam.work/api/v1/connect/*YOUR_ID*/room-manager/${roomName}/users/${userName}`
```

Comment:
**issue:** Should this use query params instead?

Reply:
Good catch. I also updated domain names from `cloud.fishjam.work` to `fishjam.io` in few places

Reply:
yes, we could already move on to query params convention